### PR TITLE
Remove focus on e-commerce

### PR DESCRIPTION
**Pull request by greg0ire**
The admin part of the Sonata project is far more active than the
e-commerce part, I think this description is more accurate.

### CODE REVIEW

File: projects/en/sonataproject.html.twig [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
-        The goal of the Sonata Project is to provide an e-commerce framework
-        solution based on the Symfony framework. Think of it as a toolkit based
-        on Symfony components to make building e-commerce sites easy and fun!
+        The goal of the Sonata Project is to provide a set of high-level features
+        built on top of the Symfony framework. Think of it as a toolkit based
+        on Symfony components to make building sites easy and fun!
         Famous bundles are already available: Admin Bundle, Block Bundle, Media
-        Bundle, ... and used by others projects like the Symfony CMF.
+        Bundle, … and used by others projects like the Symfony CMF.
```
Comment (OskarStark) [inline projects/en/sonataproject.html.twig]: used by other without s

File: projects/en/sonataproject.html.twig [hunk 2]
```diff
@@ -1,11 +1,13 @@
 {% block description %}
     <p>
 
-        The goal of the Sonata Project is to provide an e-commerce framework
-        solution based on the Symfony framework. Think of it as a toolkit based
-        on Symfony components to make building e-commerce sites easy and fun!
+        The goal of the Sonata Project is to provide a set of high-level features
+        built on top of the Symfony framework. Think of it as a toolkit based
```
Comment (fabpot) [inline projects/en/sonataproject.html.twig]: This new sentence does not say much about what Sonata is. What about focusing on the admin part instead?

Comment (greg0ire) [inline projects/en/sonataproject.html.twig]: You're right, it's the key selling point. Pushed a new version. @Soullivaneuh @core23 @OskarStark @jordisala1991 @rande, feedback is welcome.

File: projects/en/sonataproject.html.twig [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
-        solution based on the Symfony framework. Think of it as a toolkit based
-        on Symfony components to make building e-commerce sites easy and fun!
+        The goal of the Sonata Project is to provide a set of high-level
+        features built on top of the Symfony framework, with a strong focus on
+        the administrative part of things.
+        Think of it as a toolkit based on Symfony components to make building
+        sites easy and fun!
         Famous bundles are already available: Admin Bundle, Block Bundle, Media
-        Bundle, ... and used by others projects like the Symfony CMF.
+        Bundle, … and used by other projects like the Symfony CMF.
```
Comment (core23) [inline projects/en/sonataproject.html.twig]: What about the Notification (700k downloads) and Seo Bundle (500k downloads)?

Comment (greg0ire) [inline projects/en/sonataproject.html.twig]: @core23 : I added something about them below.

Comment (fabpot) [inline projects/en/sonataproject.html.twig]: Can you switch back to `...` (three dots)?

File: projects/en/sonataproject.html.twig [hunk 4]
```diff
@@ -1,11 +1,17 @@
 {% block description %}
     <p>
 
-        The goal of the Sonata Project is to provide an e-commerce framework
-        solution based on the Symfony framework. Think of it as a toolkit based
-        on Symfony components to make building e-commerce sites easy and fun!
+        The goal of the Sonata Project is to provide a set of high-level
+        features built on top of the Symfony framework, with a strong focus on
+        the administrative part of things.
```
Comment (jordisala1991) [inline projects/en/sonataproject.html.twig]: `of things` sounds strange, maybe it is better to say `of your site`?

File: projects/en/sonataproject.html.twig (hunk 3 above)
Comment (greg0ire) [inline projects/en/sonataproject.html.twig]: To be consistent? Ok… :(

File: projects/en/sonataproject.html.twig (hunk 4 above)
Comment (greg0ire) [inline projects/en/sonataproject.html.twig]: Doesn't sound right either IMO.

Comment (jordisala1991) [inline projects/en/sonataproject.html.twig]: `of your application`?

Comment (greg0ire) [inline projects/en/sonataproject.html.twig]: Looks better?

**Comment by OskarStark** [comment]
:+1:

**Comment by javiereguiluz** [comment]
It's merged now! Thanks for improving this ... and thanks to all the Sonata team for your tireless work!!

**Pull request merged**